File: ios/Components/RCA11yFocusWrapperManager/RCA11yFocusWrapperManager.m
//
//  RCTKeyboardFocusWrapper.m
//  A11y
//
//  Created by Artur Kalach on 02.09.2022.
//  Copyright © 2022 Facebook. All rights reserved.
//

#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(RCA11yFocusWrapperManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(targetField, NSNumber)
RCT_EXPORT_VIEW_PROPERTY(onFocusChange, RCTBubblingEventBlock)
RCT_EXPORT_VIEW_PROPERTY(canBeFocused, BOOL)

@end

File: ios/Components/RCA11yFocusWrapperManager/RCA11yFocusWrapperManager.swift
//
//  RCTKeyboardFocusWrapperManager.swift
//  A11y
//
//  Created by Artur Kalach on 02.09.2022.
//  Copyright © 2022 Facebook. All rights reserved.
//

import Foundation

@objc (RCA11yFocusWrapperManager)
class RCA11yFocusWrapperManager: RCTViewManager {
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
  
  override func view() -> UIView! {
    return RCA11yFocusWrapper()
  }
}
